modules/social_features/social_profile/src/Plugin/views/field/ProfileEntitySortable.php

```diff
@@ -125,8 +125,24 @@ public function clickSort($order) {
           // We will have different expressions depending on is the Nickname
           // field provided or not.
           $field = in_array('field_profile_nick_name', $order_by_fields) ?
-            "CASE WHEN profile__field_profile_nick_name.field_profile_nick_name_value IS NULL THEN CONCAT(TRIM(profile__field_profile_first_name.field_profile_first_name_value), ' ', TRIM(profile__field_profile_last_name.field_profile_last_name_value)) ELSE TRIM(profile__field_profile_nick_name.field_profile_nick_name_value) END" :
-            "CONCAT(TRIM(profile__field_profile_first_name.field_profile_first_name_value), ' ', TRIM(profile__field_profile_last_name.field_profile_last_name_value))";
+            "CASE WHEN
```

Comment:
Please provide more inline documentation with an explanation of what we are doing here.

Reply:
Provided.